app/controllers/api_controller.rb

```diff
@@ -192,4 +192,14 @@ def api_call_timeout(&block)
     ActiveRecord::Base.connection.raw_connection.cancel
     raise OSM::APITimeoutError
   end
+
+  ##
+  # check the api change rate limit
+  def check_rate_limit(new_changes = 1)
+    max_changes = ActiveRecord::Base.connection.select_value(
```

Comment:
It would be good if we could enable/disable this check based on some global setting. 

This way we could merge the code sooner and enable it on the dev instance for testing.